### DISCUSSION TITLE
Center number different from progress percentage

### DISCUSSION THREAD

**Discussion by rutenio** [tomik23/circular-progress-bar#77] [Q&A] [answered]
Is it possible for the center number to be a different value than the percentage of the circle? For example, I would like to use it as a card to inform that 10 processes are at 75%

**Comment by tomickigrzegorz** [comment, marked as answer]
You can turn off everything inside ["number": false] and add your own text or picture as in "TEXPOSITON" https://tomik23.github.io/circular-progress-bar/

You can also use the "UNIT" example, hide the pie-percent-X element with styles, and put whatever you want in the ["unit": "PL"] field.
